packages/buffetjs-styles/src/components/Button/index.js

```diff
@@ -62,18 +63,24 @@ const Button = styled.button`
       }
     `}
 
-  ${({ color }) =>
-    color === 'delete' &&
-    `
-      padding: 0 ${sizes.button.padding.small};
-      
-    `}
-
   ${({ color }) =>
     color === 'success' &&
     `
-      min-width: ${sizes.button.minWidth.small};
+      min-width: ${sizes.button.minWidth};
     `}
+  
+    // FontAwesome icons
```

Comment:
Can you comment it as follows: `/* my comment */` so we won't have to modify this file when we add stylelint?